client/src/components/MyProfile/SpeakerEditPrivateInfo.jsx

```diff
@@ -0,0 +1,217 @@
+import React, {Component} from 'react';
```

Comment:
NOTICE this file is `jsx` while all the others are `js` !
I cant find it in the dev tools :( 

![Edit_Serge_Lerner_·_Reversim_Summit_2020](https://user-images.githubusercontent.com/1729783/77056088-9f065200-69da-11ea-8ca8-d8d3111b205f.png)

Reply:
Ohh its not even in use ...

![SpeakerEditPrivateInfo_js_—_reversim-summit-2020](https://user-images.githubusercontent.com/1729783/77056496-281d8900-69db-11ea-95f2-cb70566a8803.png)

Reply:
Please delete it if tis not in use